statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
@jreback maybe now that you see this and that ``LooseVersion`` / ``StrictVersion`` are broken, you see the value in adding back ``version.short_version``? ``short_version`` is trivial to parse, while the class linked above is 130 LoC.